proposed/http-message.md

```diff
@@ -1533,6 +1533,11 @@ interface UriInterface
      * rootless (not starting with a slash). Implementations MUST support all
      * three syntaxes.
      *
+     * If the path is intended to be domain-relative rather than path relative then
```

Comment:
I wouldn't change the terminology here again. We used "host" instead of domain everywhere else. Also the domain doesn't include the port.